core/templates/pages/contact-page/contact-page-root.component.ts

```diff
@@ -17,9 +17,34 @@
  */
 
 import { Component } from '@angular/core';
+import { AppConstants } from 'app.constants';
+import { MetaAttribute, MetaTagCustomizationService } from 'services/contextual/meta-tag-customization.service';
+import { PageTitleService } from 'services/page-title.service';
 
 @Component({
   selector: 'oppia-contact-page-root',
   templateUrl: './contact-page-root.component.html'
 })
-export class ContactPageRootComponent {}
+export class ContactPageRootComponent {
+  constructor(
+    private pageTitleService: PageTitleService,
+    private metaTagCustomizationService: MetaTagCustomizationService
+  ) {}
+
+  ngOnInit(): void {
+    let pageData = AppConstants.PAGES_REGISTERED_WITH_FRONTEND.CONTACT;
+    // Update default title.
+    this.pageTitleService.setPageTitle(pageData.TITLE);
+
+    let metaAttributes: MetaAttribute[] = [];
+    for (let i = 0; i < pageData.META.length; i++) {
+      metaAttributes.push({
+        propertyType: pageData.META[i].PROPERTY_TYPE,
+        propertyValue: pageData.META[i].PROPERTY_VALUE,
+        content: pageData.META[i].CONTENT
+      });
+    }
+    // Update meta tags.
+    this.metaTagCustomizationService.addOrReplaceMetaTags(metaAttributes);
```

Comment:
We do similar operations in every migrated page, should we separate it into a function in some service? Especially the meta-tag section.

Reply:
done